userspace/libsinsp/examples/util.cpp

```diff
@@ -132,7 +132,8 @@ std::string get_event_type(uint16_t type)
         case PPME_SYSCALL_OPENAT_E:
         case PPME_SYSCALL_OPENAT_2_E:
         case PPME_SYSCALL_OPENAT_X:
-        case PPME_SYSCALL_OPENAT_2_X: return "openat";
+        case PPME_SYSCALL_OPENAT_2_X:
+        case PPME_SYSCALL_OPENAT2_X: return "openat";
```

Comment:
Shouldn't these be `openat2`?

Reply:
Great catch! I fixed this in the latest commit.